streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -91,6 +91,7 @@
     // includes assigned & initialized tasks and unassigned tasks we locked temporarily during rebalance
     private final Set<TaskId> lockedTaskDirectories = new HashSet<>();
     private java.util.function.Consumer<Set<TopicPartition>> resetter;
+    private Map<TopicPartition, Long> committedOffsets = new HashMap<>();
```

Comment:
During a rebalance, we should delete all entries for partitions we don't own any longer.

Should we also pre-populate this map when we init a task (cf `StreamsTask#initializeMetadata()`)